### ISSUE TITLE
Variable Fonts: font and file names

### ISSUE THREAD

**Issue by vv-monsalve** [petrvanblokland/TYPETR-Bitcount#9]
The current filenames are:

````
Bitcount-Color[wght,ELXP,ELSH,slnt,SZP1,XPN1,YPN1,SZP2,XPN2,YPN2].ttf
BitcountGridDouble-Color[wght,ELXP,ELSH,slnt,SZP1,XPN1,YPN1,SZP2,XPN2,YPN2].ttf
BitcountGridDouble[wght,ELXP,ELSH,slnt].ttf
BitcountGridSingle-Color[wght,ELXP,ELSH,slnt,SZP1,XPN1,YPN1,SZP2,XPN2,YPN2].ttf
BitcountGridSingle[wght,ELXP,ELSH,slnt].ttf
BitcountPropDouble-Color[wght,ELXP,ELSH,slnt,SZP1,XPN1,YPN1,SZP2,XPN2,YPN2].ttf
BitcountPropDouble[wght,ELXP,ELSH,slnt].ttf
BitcountPropSingle-Color[wght,ELXP,ELSH,slnt,SZP1,XPN1,YPN1,SZP2,XPN2,YPN2].ttf
BitcountPropSingle[wght,ELXP,ELSH,slnt].ttf
BitcountSingle-Color[wght,ELXP,ELSH,slnt,SZP1,XPN1,YPN1,SZP2,XPN2,YPN2].ttf
BitcountSingle[wght,ELXP,ELSH,slnt].ttf
Bitcount[wght,ELXP,ELSH,slnt].ttf
````

Please review the following
- General requirements for [font names](https://googlefonts.github.io/gf-guide/requirements.html#font-name) 
    - Hyphens and the "Color" word are not allowed as font or file names. 
- And the specifi VF requirements for [file names](https://googlefonts.github.io/gf-guide/variable.html#variable-font-filenames)
    - The custom axes should go first, listed in alphabetical order (but wght always last)

So, for example, after inspectig the above first font in detail, it should be

````
BitcountMonoDouble[ELSH,ELXP,SZP1,SZP2,XPN1,XPN2,YPN1,YPN2,slnt,wght].ttf
````

**Comment by simoncozens** [comment]
The axes naming is fixed in #13. What should we use instead of Color?

**Comment by simoncozens** [comment]
Actually if we are only onboarding the color, then we don't need to distinguish the two builds and we are done here.
